bulbs/instant_articles/templates/instant_article/base_instant_article.html

```diff
@@ -38,7 +38,7 @@ <h3 class="op-kicker">
 
         <figure>
           {% block cover %}
-            <img src="{% cropped_url content.detail_image.id %}" />
+            <img src="{% cropped_url content.detail_image.id fixed=True %}" />
```

Comment:
just added: https://trello.com/c/6dygMdoH/120-django-betty-cropper-rename-fixed-and-betty-fixed-url